### DISCUSSION TITLE
Add shadcnui and v0 compatibility out of the box on nextjs templates

### DISCUSSION THREAD

**Discussion by Sign0ret** [payloadcms/payload#4636] [Feature Requests & Ideas]
would be amazing to have this integrations on default, since the shadcnui components are amazing but not compatible with the actual config since it uses scss and not tailwind.

**Comment by 99power** [comment]
> would be amazing to have this integrations on default, since the shadcnui components are amazing but not compatible with the actual config since it uses scss and not tailwind.

Did you get around using Tailwind at all? I could see how for custom components in the admin UI that makes sense, but for the frontend when we're dynamically creating elements, how would that work in your opinion?

**Reply by revnelson** [reply]
Check out this thread: https://github.com/payloadcms/payload/discussions/845
